src/modules/core/components/DetailsWidgetUser/DetailsWidgetUser.tsx

```diff
@@ -23,11 +24,16 @@ const DetailsWidgetUser = ({ walletAddress }: Props) => {
   return (
     <div className={styles.main}>
       <UserAvatar
+        colony={colony}
         size="s"
         notSet={false}
         user={userProfile}
         address={walletAddress || ''}
         showInfo
+        popperProps={{
+          showArrow: false,
+          placement: 'bottom',
+        }}
```

Comment:
I think this looks better with a `left` placement and a bit of offset

```suggestion
        popperProps={{
          showArrow: false,
          placement: 'left',
          modifiers: [
            {
              name: 'offset',
              options: {
                offset: [0, 10],
              },
            },
          ],
        }}
```

![Screenshot from 2021-12-03 11-28-52](https://user-images.githubusercontent.com/1193222/144579115-b1e28520-6711-4dd4-8774-6ea52b4efec5.png)